### PR TITLE
fix task MinAvailable issue

### PR DESCRIPTION
**Pull request by merryzhou**
1. task MinAvailable should greater than 0 when create & update job
2. skip check when task MinAvailable == 0

Signed-off-by: merryzhou <merryzhou0882@gmail.com>

### CODE REVIEW

File: pkg/scheduler/api/job_info.go [hunk 1]
```diff
@@ -704,6 +704,9 @@ func (ji *JobInfo) CheckTaskMinAvailable() bool {
 
 	klog.V(4).Infof("job %s/%s actual: %+v, ji.TaskMinAvailable: %+v", ji.Name, ji.Namespace, actual, ji.TaskMinAvailable)
 	for task, minAvailable := range ji.TaskMinAvailable {
+		if minAvailable <= 0 {
```
Comment (hwdef) [inline pkg/scheduler/api/job_info.go:707]: Why not intercept invalid values ​​via webhook?

Comment (Thor-wl) [reply pkg/scheduler/api/job_info.go:707]: Why can it be an invalid value? I agree with @hwdef that it should be checked in the webhook.

Comment (merryzhou) [reply pkg/scheduler/api/job_info.go:707]: when task replica == 0，then minAvailable will be zero.

and is valid to set task replica to 0

Comment (Thor-wl) [reply pkg/scheduler/api/job_info.go:707]: /cc @william-wang How do you think about it? Is there some scenarios that users may set `task.replicas` to be `0`?

Comment (william-wang) [reply pkg/scheduler/api/job_info.go:707]: task.Replicas is allowed to be `0`.  Please do this kind of check in webhook.

Comment (merryzhou) [reply pkg/scheduler/api/job_info.go]: done

**Pull request merged**